ledger-tool/src/main.rs

```diff
@@ -2962,6 +3045,32 @@ fn main() {
             ));
             println!("Ok.");
         }
+        ("compute-cost", Some(arg_matches)) => {
+            let blockstore = open_blockstore(
+                &ledger_path,
+                AccessType::TryPrimaryThenSecondary,
+                wal_recovery_mode,
+            );
+            let allow_dead_slots = arg_matches.is_present("allow_dead_slots");
```

Comment:
I think we can remove the `allow_dead_slots` argument entirely, and just error in `compute_cost_by_slot` if the slot is dead

Reply:
make sense, removed it